FirebaseMessaging/Tests/UnitTests/FIRMessagingCheckinStoreTest.m

```diff
@@ -79,23 +85,22 @@ - (void)testInvalidCheckinPreferencesOnKeychainFail {
       expectationWithDescription:@"Checkin preference should be invalid after keychain failure"];
 
   FIRMessagingFakeKeychain *fakeKeychain = [[FIRMessagingFakeKeychain alloc] init];
-  FIRMessagingCheckinStore *checkinStore = [[FIRMessagingCheckinStore alloc] init];
-  checkinStore.keychain = fakeKeychain;
+  self.checkinStore.keychain = fakeKeychain;
   __block FIRMessagingCheckinPreferences *preferences =
       [[FIRMessagingCheckinPreferences alloc] initWithDeviceID:kAuthID secretToken:kSecret];
   [preferences updateWithCheckinPlistContents:[[self class] newCheckinPlistPreferences]];
-  [checkinStore saveCheckinPreferences:preferences
-                               handler:^(NSError *error) {
-                                 XCTAssertNil(error);
-                                 fakeKeychain.cannotReadFromKeychain = YES;
-                                 preferences = [checkinStore cachedCheckinPreferences];
-
-                                 XCTAssertNil(preferences.deviceID);
-                                 XCTAssertNil(preferences.secretToken);
-                                 XCTAssertFalse([preferences hasValidCheckinInfo]);
-
-                                 [checkinInvalidExpectation fulfill];
-                               }];
+  [self.checkinStore saveCheckinPreferences:preferences
+                                    handler:^(NSError *error) {
+                                      XCTAssertNil(error);
+                                      fakeKeychain.cannotReadFromKeychain = YES;
+                                      preferences = [self->_checkinStore cachedCheckinPreferences];
```

Comment:
nit: use `self.checkinStore` to be consistent with access above

Reply:
This is to avoid implicitly retain 'self' warning: https://stackoverflow.com/questions/21577711/block-implicitly-retains-self-explicitly-mention-self-to-indicate-this-is-i?noredirect=1&lq=1